extensions/wikidata/src/org/openrefine/wikidata/qa/WikidataConstraintFetcher.java

```diff
@@ -68,9 +71,27 @@
 
     public static String MANDATORY_QUALIFIERS_CONSTRAINT_QID = "Q21510856";
     public static String MANDATORY_QUALIFIERS_CONSTRAINT_PID = "P2306";
+    
+    public static String ALLOWED_VALUES_CONSTRAINT_QID = "Q21510859";
+    public static String ALLOWED_VALUES_CONSTRAINT_PID = "P2305";
+
+    public static String DISALLOWED_VALUES_CONSTRAINT_QID = "Q52558054";
+    public static String DISALLOWED_VALUES_CONSTRAINT_PID = "P2305";
 
     public static String SINGLE_VALUE_CONSTRAINT_QID = "Q19474404";
+    public static String SINGLE_BEST_VALUE_CONSTRAINT_QID = "Q52060874";
     public static String DISTINCT_VALUES_CONSTRAINT_QID = "Q21502410";
+    
+    public static String NO_BOUNDS_CONSTRAINT_QID = "Q51723761";
+    public static String INTEGER_VALUED_CONSTRAINT_QID = "Q52848401";
+    
+    public static String ALLOWED_UNITS_CONSTRAINT_QID = "Q21514353";
+    public static String ALLOWED_UNITS_CONSTRAINT_PID = "P2305";
+    
+    public static String ALLOWED_ENTITY_TYPES_QID = "Q52004125";
```

Comment:
that is planned - this will be part of making the extension usable for any Wikibase. But that's a lot more work and I don't think it should be ready for 3.0.